tests/test_logic.py

```diff
@@ -15,7 +15,7 @@ def setUp(self):
         self.bot = Bot()
         self.logic = BotLogic(self.bot)
 
-    def testParsePrivMsg(self):
+    def test_ParsePrivMsg(self):
```

Comment:
I think we should have underscore separators everywhere so this would be test_parse_priv_msg(